docs/jwst/jump/description.rst

```diff
@@ -96,3 +96,17 @@ The use of the reference files is flexible. Full-frame reference
 files can be used for all science exposures, in which case subarrays will be
 extracted from the reference file data to match the science exposure, or
 subarray-specific reference files may be used.
+
+Large Events (Snowballs and Showers)
+====================================
+All the detectors on JWST are effected by large cosmic ray
+events. While these events, in general, effect a large number of
```

Comment:
same here: should be "affected", which is the verb version; "effect" is the noun version  (e.g. the snowball effect)